src/styles/blocks/_catalog.scss

```diff
@@ -0,0 +1,37 @@
+.catalog {
+  display: grid;
+  grid-template-columns: repeat(1, $base-card-width);
+  justify-content: center;
+  gap: 0 48px;
+
+  margin: 0 auto;
+
+  &__card {
+    display: grid;
+    grid-template-columns: repeat(1, $base-card-width);
+    text-align: center;
+    align-content: center;
+    height: 300px;
+    border: 1px solid #ccc;
+    border-radius: 5px;
+    margin-bottom: 48px;
```

Comment:
Remove the margin-bottom from the catalog card as it's not needed due to grid gap.